src/components/Card.js

```diff
@@ -1,19 +1,30 @@
 import styled from 'styled-components';
+import { BsTrash } from 'react-icons/bs';
 
-export default function Card({ word, example, explanation }) {
+export default function Card({
+  word,
+  example,
+  explanation,
+  onDeleteCard,
+  _id,
+}) {
   return (
     <CardWrapper>
       <CardWord>{word}</CardWord>
       <CardExample>{example}</CardExample>
       <CardExplanation>{explanation}</CardExplanation>
```

Comment:
Maybe it's more semantic to use a `dl` with one `dt` (word) and two `dd`s (example and explanation) here?
Or I would at least set the word as a heading of the card.

Reply:
you're right. I will change it according to your suggestion. Thank you a lot!